scripts/install-git-secrets.sh

```
@@ -0,0 +1,83 @@
+# Install git-secrets hooks in the repository
+echo "🔧 Installing git-secrets hooks..."
+git secrets --install -f
+
+# Add common patterns to detect
+echo "📝 Adding secret patterns..."
+
+# AWS patterns
+git secrets --register-aws
+
+# Additional custom patterns
+git secrets --add 'api[_-]?key.*[:=]\s*["\'][^"\']+["\']' || true
```

Shellcheck (0.10.0)

[warning] Did you forget to close this single quoted string?

(SC1078)